README.md

```diff
@@ -26,6 +34,8 @@ make build
 make run
 ```
 
+**Note** You will need to have the database and Consul running before you execute `make run`. If you don't want to install them, start them via their respective Docker containers.
```

Comment:
The services can be run without Consul. They only require Consul if they are started with the `--consul` flag which is not the default via `make run`

Reply:
Thanks for reviewing. Corrected.